postmortem.md

```diff
@@ -15,24 +29,172 @@ Este documento é uma reflexão do time, realizada na fase de finalização do p
+- _Pontos Negativos_: A comunicação em geral foi algo que precisavamos melhorar muito no trabalho, principalmente na reta final com o desfoco de alguns membros devido ao aumento da carga horaria em otras materias. Falta de alinhamento o que impactou na diminuição do escopo do projeto.
 
 # Tecnologias utilizadas
 
 ## Tecnologias utilizadas no backend
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis massa purus, finibus vel massa non, consectetur congue neque. In hac habitasse platea dictumst. Nam dolor est, condimentum in efficitur et, sagittis non sem. Nunc sed libero tempor, ornare lacus at, fringilla ante. Sed luctus eros nec purus commodo, vel maximus ipsum convallis. Aenean et nisi vitae nibh faucibus aliquam. Proin egestas nec tortor at blandit. Aenean mauris ex, tincidunt et imperdiet eu, mattis auctor eros.
+Para desenvolvimento do backend da aplicação, foi utilizado Node.js. Com relação ao banco de dados, foi escolhido o MongoDB Atlas devido à sua facilidade de integração com a linguagem citada anteriormente e por ser salvo em nuvem.
 
-## Tecnologias utilizadas no frontend
+Dentre os módulos de Node.js utilizados, vale destacar:
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis massa purus, finibus vel massa non, consectetur congue neque. In hac habitasse platea dictumst. Nam dolor est, condimentum in efficitur et, sagittis non sem. Nunc sed libero tempor, ornare lacus at, fringilla ante. Sed luctus eros nec purus commodo, vel maximus ipsum convallis. Aenean et nisi vitae nibh faucibus aliquam. Proin egestas nec tortor at blandit. Aenean mauris ex, tincidunt et imperdiet eu, mattis auctor eros.
+- Express: Criação de servidores e requisições.
+- Mongoose: Conexão com o banco de dados e criação de models.
+- Jest e Supertest: Criação de testes unitários.
+
+## Tecnologias utilizadas no mobile
+
+Para desenvolvimento mobile, a equipe utilizou do framework flutter juntamente de emuladores (IOS e Android). Para os emuladores no android foi necessário baixar o android studio.
+Vale ressaltar o uso de pacotes de grande relevância para nossa aplicação são eles:
+
+- Retrofit: Geração de código para cliente dio.
+- Dio: Realização de requisições HTTP.
+- Freezed: Geração de código para classes (models) e para padrão BLoC.
+- BLoC: Gerenciamento de estados da aplicação.
+- Equatable: Auxilia na escrita do override de operadores de comparação
+- Get_It: Localizador de serviços.
+- Flutter_rating_bar: Estrelas para geração de avaliações.
 
 ## Tecnologias utilizadas na configuração dos ambientes
 
+- Desenvolvimento em sistemas operacionais unix (MacOs e Linux).
```

Comment:
I think that would be better "Desenvolvimento nos sistemas operacionais MacOS e Linux."